HARSHAL/Reactjs/react_app/src/components/assignment2.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+
+export default class IncrementDecrement extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            count: 0,
+        };
+    }
+    changecountincrement = () => {
+        this.setState({
+            count: this.state.count + 1,
+        });
+        if(this.state.count>9) {alert("value greater than 10")}
+    };
+    changecountdecrement = () => {
+        this.setState({
```

Comment:
counter value must not be decremented after 0
and not be incremented after 10
put proper conditons